EOS-uiux-Solutions/user-story
# Okta optional login

Signed-off-by: Hrishikesh Agarwal <hrishikeshagarwalv@gmail.com>

**Issue Number**

fixes #266 



**Describe the changes you've made**

Added Okta login button on the main authentication page.


**Additional context (OPTIONAL)**

![screencapture-localhost-3000-login-1665530234143](https://user-images.githubusercontent.com/54779517/195215788-f835f449-8fb5-4c34-9053-98baaedf47b4.png)


**Test plan (OPTIONAL)**

1. Set SSO to true in config.json
2. Then click on the sign-in button on the home page.
3. Check the okta login button



**Checklist**
